### ISSUE TITLE
Randomizer causes game to crash/hang

### ISSUE THREAD

**Issue by LordHDL** [Lakifume/True-Randomization#40]
The randomizer just seems to cause crashes and hangs for me every time. It happens either when loading the game or when selecting a character (just before where you normally choose a difficulty). I get a window that says "Fatal error" and I have to end task. I've tried on both 1.5 and 1.4, with and without save files, with and without cloud saving on, and repeatedly generated different seeds. Nothing works. This is on the GoG version without DLC installed and version 3.0.3 of the randomizer.

**Comment by Lakifume** [comment]
Try deleting the NoAurora folder in Bloodstained Ritual of the Night\BloodstainedRotN\Binaries\Win64\Mods and see if that fixes it.

**Comment by LordHDL** [comment]
No good. Doesn't seem to have any noticeable effect.

**Comment by Lakifume** [comment]
Okay, in that case try deleting the entire Mods folder in Bloodstained Ritual of the Night\BloodstainedRotN\Binaries\Win64

**Comment by LordHDL** [comment]
Same thing, still says fatal error. Although this time the dialogue box that appears when launching the game complains that the Mods folder is gone.

**Comment by Lakifume** [comment]
Okay are you sure you're able to load the game at all, even without any mods on ?

**Comment by LordHDL** [comment]
I can run the game with no issues after deleting the extra stuff the randomizer puts into Binaries/Win64 and Content/Paks. I don't have any mods. I've tried uninstalling and verifying game files as well but that doesn't seem to be a factor. I've been trying mostly with version 1.4 of the game since your description on nexusmods says the randomizer is for 1.4, but I've tried with 1.5 as well.

**Comment by Lakifume** [comment]
Oh yeah it's meant for 1.5, forgot to update the text on Nexus. Okay try with just the rando pak file alone

**Comment by LordHDL** [comment]
All right I was able to actually load a seed finally, but I still get the fatal error window popup. The game seems to run (for now) if I ignore it, but if I click "ok" on the window the game closes. I've also noticed a lot of the seeds I generate appear as unbeatable when I click "test seed." Is that currently expected?

**Comment by LordHDL** [comment]
I have map rando enabled (not using any custom ones). As for the fatal error message, the game hangs if I set that option to false. Leaving it set to true allows me to get in-game. The message appears during the intro video (the one that shows 505 Games and ArtPlay).

Also I noticed that playing with randomizer causes a weird framerate stutter on occasion that doesn't happen in vanilla. Not sure if that's known so I figured I'd mention it.

**Comment by Lakifume** [comment]
Unfortunately you're the only one having all these issues atm, I've ran out of ideas.

**Comment by LordHDL** [comment]
It's all right, you helped me to at least get it working so I can load a proper seed now. Thanks.

**Comment by LordHDL** [comment]
I figured out how to avoid the fatal error message and crashing/hanging. Turns out it only happens when launching the game through the Galaxy client. If launching the game's exe directly it runs without issue. My guess is that the overlay or stat tracking from the client is not playing nice with the randomizer (or some file path weirdness). In any case, for those who use the GoG version, launch the game application directly to avoid this problem.
